homepage/index.html

```diff
@@ -0,0 +1,214 @@
+                    <option value="ACCESS 103">ACCESS 103</option>
+                </select>
+            </form>
+            <button>
+                <p>Send me the Layout</p>
+            </button>
+        </div>
+    </div>
+
+    <div class="online">
+        <div class="flex1">
+            <span>Online or Offline?</span>
+            <p>The only difference between our Online and Offline classes is where you’re located. Both options provide:
+            </p>
+            <ul>
+                <li>Live, Real-Time Instruction</li>
+                <li>Scheduled Lectures and Projects</li>
+                <li>Direct access with Mentors</li>
+                <li>Access Study Material</li>
+                <li>Community Networking</li>
+                <li>Career Support</li>
+            </ul>
+            <div class="online-text">
+                <p>If you’d like to know more—call, email, or chat with us.</p>
+            </div>
+            <button>
+                <p>Talk to Admissions</p>
+            </button>
+        </div>
+        <div class="flex2">
+
+            <img src="/homepage/images/Group 328.png" alt="">
+        </div>
+    </div>
+
+
+    <div class="covid">
+        <p>Due to COVID we are only offereing ONLINE Classes</p>
+    </div>
+
+    <div class="apply">
+        <span>How to Apply?</span>
+        <p>Attend the DEMO Class a week prior of starting of every batch. If satisfied with the approach of the mentors,
+            then Apply for the course. <br>
+            <br>
+            Applying is the first step toward attending access programme and changing or starting your tech career.
+            Learn about the coding bootcamp application process and speak to our admissions counselor.
+        </p>
+        <button>
+            <p>Details</p>
+        </button>
+    </div>
+
+
+    <div class="career-support">
+        <p>Career Support Programme</p>
+        <div class="container">
+            <div class="image-container"><img src="/homepage/images/career support.png" alt=""></div>
+            <div class="text-container">
+                <p>Let’s walk together in your toughest time. After the completion of course, you shall be looking for
+                    your dream jobs. Let us help you reach your destination, with our career support programme</p>
+                <button><span>Details</span></button>
+            </div>
+        </div>
+    </div>
+
+    <div class="faizan">
+        <span>Why did I choose access!</span>
+        <div class="faizan-flex">
+            <div class="faizan-image"><img src="/homepage/images/faizan.png" alt=""></div>
+            <div class="faizan-text">
+                <p>
+                    The best part of being with access, is that you are constantly learning and upgrading yourself to
+                    the
+                    industry standards.
+                </p>
+                <div class="just-dont"><span>Faizan Khan | Web Developer</span></div>
```

Comment:
This should come in single line 

<img width="149" alt="Screenshot 2021-09-17 at 9 28 19 AM" src="https://user-images.githubusercontent.com/47788183/133721936-8ad70017-e56a-412f-9949-44a7e95fbf1a.png">

Reply:
done